yarn.lock

```diff
@@ -4,7 +4,7 @@
 
 "@discordjs/builders@^0.11.0":
   version "0.11.0"
-  resolved "https://registry.yarnpkg.com/@discordjs/builders/-/builders-0.11.0.tgz#4102abe3e0cd093501f3f71931df43eb92f5b0cc"
+  resolved "https://registry.npmjs.org/@discordjs/builders/-/builders-0.11.0.tgz"
```

Comment:
Can you revert the changes to `yarn.lock` too? This looks like it might have been because of a version change or something, but I don't think it's necessary right now

Reply:
I don't know anything about it. What should I do ????

Reply:
@nickmccurdy whats your opinion ???

Reply:
I don't know anything about it, what should I do ???

Reply:
You should be able to do something like `git checkout main -- yarn.lock`, this has more info https://dev.to/lofiandcode/git-and-github-how-to-revert-a-single-file-dha

Reply:
done !